### PR TITLE
es: CR2 Chromium translations

### PR DESCRIPTION
**Pull request by henrif75**
Professional translations for CR2 content.
Changes are being staged in a separate branch until all Spanish content is reviewed.

#1463 #284

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -14403,35 +14671,48 @@ msgid ""
 "missing in the `png` crate - e.g. RGBA => BGRA, or gamma correction. Such "
 "functionality may benefit from separate tests authored in Rust."
 msgstr ""
+"La integración hipotética de PNG y WIP puede necesitar una implementación "
```
Comment (deavid) [inline po/es.po:14674]:
```suggestion
"La integración hipotética/WIP de PNG puede necesitar una implementación "
```

File: po/es.po [hunk 2]
```diff
@@ -14505,55 +14825,71 @@ msgid ""
...
 
 #: src/exercises/chromium/testing.md
 msgid ""
 "Add a testable function next to `hello_from_rust`. Some suggestions: adding "
 "two integers received as arguments, computing the nth Fibonacci number, "
 "summing integers in a slice, etc."
 msgstr ""
+"Añade una función que se pueda probar junto a `hello_from_rust`. Aquí tienes "
+"algunas sugerencias: añadir dos números enteros recibidos como argumentos, "
+"calcular n en la serie de Fibonacci, sumar los enteros en un slice, etc."
```
Comment (deavid) [inline po/es.po:14855]:
```suggestion
"calcular el enésimo número de la serie Fibonacci, sumar los enteros en un slice, etc."
```

File: po/es.po [hunk 3]
```diff
@@ -15520,64 +16143,83 @@ msgid ""
...
 
 #: src/chromium/adding-third-party-crates/checking-in.md
 msgid "Checking Crates into Chromium Source Code"
-msgstr ""
+msgstr "Comprobar crates en el código fuente de Chromium"
 
 #: src/chromium/adding-third-party-crates/checking-in.md
 msgid "`git status` should reveal:"
-msgstr ""
+msgstr "`git status debe revelar lo siguiente:"
```
Comment (deavid) [inline po/es.po:16190]:
```suggestion
msgstr "`git status` debe revelar lo siguiente:"
```

**Review by deavid — CHANGES_REQUESTED** [review]

**Comment by henrif75** [comment]
@deavid Can you check whether you were logged with your personal email (that can explain the "missing CLA")?

**Comment by deavid** [comment]
> @deavid Can you check whether you were logged with your personal email (that can explain the "missing CLA")?

I am always logged with my personal email. I don't use corp for GitHub. Moreover, all changes and reviews have been done from the same personal computer. There's no difference from my side.

I don't see a CLA problem, I see a format problem. One of the suggestions I gave you breaks the formatter. Can you fix?

**Comment by deavid** [comment]
I see now that the CLA was failing, my bad. And after my review it is still failing. Strange.

@mgeisler do you know how we can debug this? My understanding is that I don't need to sign CLA, and I did not need to sign it for anything before - the other PRs are going smoothly.

The only thing I can think of is that there's a commit here that says "co-authored-by" and has my email. Not sure if that is the problem, that maybe Google doesn't store the association on my email but to my GitHub account.

If there's need to do, I can sign the CLA, but since I was told that I shouldn't need to do it, I'm holding doing that until we understand if that's the course of action.

**Comment by mgeisler** [comment]
> @mgeisler do you know how we can debug this? My understanding is that I don't need to sign CLA, and I did not need to sign it for anything before - the other PRs are going smoothly.
>
> The only thing I can think of is that there's a commit here that says "co-authored-by" and has my email. Not sure if that is the problem, that maybe Google doesn't store the association on my email but to my GitHub account.

I think this is the problem: it's only `@google.com` addresses that are exempt for the CLA check. The check has [this link](https://goto.google.com/prinfo/github.com%2Fgoogle%2Fcomprehensive-rust%2Fpull%2F1868) to look further. Right now it says everything is fine, but that's probably because the commit was edited by Henri or you.

**Comment by deavid** [comment]
It was edited by Henri. Told him to delete the Co-Authored or replace it with deavid@google.com

It seems that for some operations GitHub might be adding the email account linked.

If it's me who is doing the commits, I can always take care and commit as deavid@google.com, but I can't control from the UI, or what others do.

**Comment by mgeisler** [comment]
Yes, you're spot on... we've seen this before with other Googlers and I don't think there is a better fix than editing the commit like this.

**Pull request merged**